diff -ur trojita-0.5.old/src/AbookAddressbook/AbookAddressbook.h trojita-0.5/src/AbookAddressbook/AbookAddressbook.h
--- trojita-0.5.old/src/AbookAddressbook/AbookAddressbook.h	2015-02-23 00:49:04.000000000 +0100
+++ trojita-0.5/src/AbookAddressbook/AbookAddressbook.h	2015-10-04 17:29:52.504149362 +0200
@@ -25,6 +25,7 @@
 #ifndef ABOOK_ADDRESSBOOK
 #define ABOOK_ADDRESSBOOK
 
+#include <QObject>
 #include <QPair>
 #include "Gui/AbstractAddressbook.h"
 
diff -ur trojita-0.5.old/src/Imap/Model/MailboxMetadata.h trojita-0.5/src/Imap/Model/MailboxMetadata.h
--- trojita-0.5.old/src/Imap/Model/MailboxMetadata.h	2015-02-23 00:49:04.000000000 +0100
+++ trojita-0.5/src/Imap/Model/MailboxMetadata.h	2015-10-04 22:46:11.139262384 +0200
@@ -25,6 +25,7 @@
 
 #include <QDebug>
 #include <QStringList>
+#include <QDataStream>
 
 namespace Imap
 {
